WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/DevicePortal.cs

```diff
@@ -172,15 +181,13 @@ public string PlatformName
         /// <param name="ssid">Optional network SSID.</param>
         /// <param name="ssidKey">Optional network key.</param>
         /// <param name="updateConnection">Indicates whether we should update this connection's IP address after connecting.</param>
-        /// <param name="rawManualCertificate">Allows specifying the raw certificate manually. This allows loading it from a local file or loading an override for a web proxy.</param>
         /// <remarks>Connect sends ConnectionStatus events to indicate the current progress in the connection process.
         /// Some applications may opt to not register for the ConnectionStatus event and await on Connect.</remarks>
         /// <returns>Task for tracking the connect.</returns>
         public async Task Connect(
             string ssid = null,
             string ssidKey = null,
-            bool updateConnection = true,
-            byte[] rawManualCertificate = null)
```

Comment:
I mostly removed it here since the UWP flow wasn't actually doing anything with it, so it felt kind of strange. I might add it back when I fix up that flow if it makes sense, but right now you accomplish the same thing with a SetDeviceCertificate on the connection followed by a Connect on the portal.